synthetic_data/synthetic_data.py

```diff
@@ -320,22 +328,15 @@ def make_tabular_data(
         tmp_norm = stats.norm(loc=x_tmp.mean(), scale=x_tmp.std())
         x_cont[:, i] = tmp_norm.cdf(x_tmp)
 
-    # norm = stats.norm()
```

Comment:
@tazitoo , on line 316-318 above, I had this removed. Do we need to do this as shouldn't the `transform_to_distribution` shift the distribution give the col_mean info when given to the dist function?

Reply:
I put it back for now, but want to ensure before we merge